spec/index.html

```diff
@@ -2282,13 +2280,29 @@ <h2>Substantive changes between RDF 1.0 and RDF 1.1 </h2>
   <h2>Substantive changes since RDF 1.1</h2>
 
   <ul>
+    <li> The major change between the RDF 1.1 and RDF 1.2 semantics is the addition of triple terms.
+      Various parts of the semantics have been updated to handle triple terms.
+    </li>
+
     <li> RDF entailment rule <a>rdfD1a</a> was added in RDF 1.2.  
       This rule should have been included in RDF 1.1 when the two built-in
       datatypes (<code>xsd:string</code> and <code>rdf:langString</code>)
       were added to RDF entailment.
     </li>
-    <li><code>rdf:dirLangString</code> added to the built-in datatypes.</li>
+
+    <li><code>rdf:dirLangString</code> was added to the built-in datatypes.</li>
+
+    <li>
+      In RDF 1.1 <code>rdf:PlainLiteral</code> was described as an optional datatype that, if recognized, 
+      MUST be interpreted to denote the datatype defined in [[RDF-PLAIN-LITERAL]].
+      <code>rdf:PlainLiteral</code> is not used elsewhere in the RDF documents,
+      so the requirement to give it this particular semantics has been removed.
+      It is recommended that <code>rdf:PlainLiteral</code> not be used in RDF.
```

Comment:
I believe this appendix is INFORMATIVE, so `MUST` (line 2297) should be made lowercase and `recommended` (line 2300) should be kept so. If this appendix is NORMATIVE, then `MUST` (line 2297) should be kept uppercase, and `recommended` (line 2300) should also be made so. In either case, the introductory `In RDF 1.1` should be followed by a comma.
```suggestion
      In RDF 1.1, <code>rdf:PlainLiteral</code> was described as an optional datatype that, if recognized, 
      must be interpreted to denote the datatype defined in [[RDF-PLAIN-LITERAL]].
      <code>rdf:PlainLiteral</code> is not used elsewhere in the RDF documents,
      so the requirement to give it this particular semantics has been removed.
      It is recommended that <code>rdf:PlainLiteral</code> not be used in RDF.
```

Reply:
> My understanding is that commas are optional after short introductory phrases.

Yes, "commas are optional after short introductory phrases", and, when omitted, they are inferred by strong/native readers of English. However, we are not writing for strong/native readers of English; we are writing for anyone on Earth. It is therefore better to include such optional commas.

> The MUST wording is from RDF 1.1, where it was in a normative section, so it should be MUST here because it was MUST there. This section is non-normative so recommended is lower case. (This sentence may be removed, however.)

As a quotation from RDF 1.1, the phrase should be wrapped in quotation marks. Optimally, there would be a citation with a [direct link to where this statement occurs](https://www.w3.org/TR/rdf11-mt/#dfn-recognize:~:text=recognized%2C%20it%20must%20be%20interpreted%20to%20refer%20to%20the%20datatype%20defined%20in%20%5BRDF%2DPLAIN%2DLITERAL%5D.). I generated that link with Chrome which may still be the only browser that creates them properly, but if my Google searching has been sufficient, most (Chrome, Safari, Edge, Firefox) if not all current browsers can *dereference* them properly. I don't know what ReSpec markup to use for such a citation, especially not if retaining the `<a ... /a>` within the quotation....

```suggestion
      In RDF 1.1, <code>rdf:PlainLiteral</code> was described as an optional datatype
      that, "[if] [recognized](https://www.w3.org/TR/rdf11-mt/#dfn-recognize), ...
      must be interpreted to [denote] the datatype defined in [[RDF-PLAIN-LITERAL]]."
      <code>rdf:PlainLiteral</code> is not used elsewhere in the RDF documents,
      so the requirement to give it this particular semantics has been removed.
      It is recommended that <code>rdf:PlainLiteral</code> not be used in RDF.
```